### ISSUE TITLE
Translations are attributed as 'reviews' in last action

### ISSUE THREAD

**Issue by iafan** [translate/pootle#4621]
Example: if one goes to https://translate.evernote.com/id/web_evernote/translate/resources/notes/onboarding/Jot_down_ideas.enex.master.po#unit=5382977 they will see that the unit was translated (there was no review, according to the log):

![screenclip 5](https://cloud.githubusercontent.com/assets/1728158/14256433/f0f967b0-fa4d-11e5-93e9-78847ea124ad.png)

TM results also suggest that there was a 100% match.

Now if one goes to extended stats and searches for this unit, they will see this:

![screenclip 6](https://cloud.githubusercontent.com/assets/1728158/14256507/3417e238-fa4e-11e5-93c1-3b17319de131.png)

Indeed the unit was translated with 100% similarity. My understanding is that the translator got to this unit, and at this moment Pootle pre-populated the translation with a fuzzy value from the TM. Translator just removed the "Needs action" flag and hit "Submit".

So far so good. But when one opens the browse UI and sees the last action for this unit, it shows this:

![screenclip 7](https://cloud.githubusercontent.com/assets/1728158/14256622/bbb53c18-fa4e-11e5-881c-d97ded5b7c0d.png)

And the action here is "reviewed", not "translated", which is misleading.

I believe this is a regression. Since the last time we updated our translation server was March, 17. Something likely happened before this date that affected the rendering of the last action snippet.

**Comment by ta2-1** [comment]
When we add translation we create a couple of submissions (for `STATE` and `TARGET` fields).
This is very likely that we can create last action message based on `STATE` field submission but we have to use `TARGET` field submission instead.
See https://github.com/translate/pootle/blob/a064d54b69d7fb344c04accf12876845850116eb/pootle/apps/pootle_statistics/models.py#L311-L313.

**Comment by julen** [comment]
When the stats cache retrieves the latest submission, it currently [determines the order by `creation_time` + `pk`](https://github.com/translate/pootle/pull/4025), and since [the change to the state is recorded after the change to the target](https://github.com/translate/pootle/blob/a064d54b69d7fb344c04accf12876845850116eb/pootle/apps/pootle_store/models.py#L1586-L1597), the latest submission information refers to the state change and therefore it interprets it as a review.

After diving deep through git log, I can confirm [this has always been the case](https://github.com/translate/pootle/commit/68fc7235ab230212138b7c26c8222c447a1f4bf7#diff-0038950c975bb50b22ceffd7badd6276R1408) and hence it's not a regression but a bug. 
Still, it's an unknown why the code wasn't adjusted at the time [the comment linked above](https://github.com/translate/pootle/blob/a064d54b69d7fb344c04accf12876845850116eb/pootle/apps/pootle_statistics/models.py#L311-L313) was added. We definitely need to fix this but as well as improve the test coverage when it comes to submissions and recorded events.

So summing up, this has always been an issue but it could as well get more noticeable recently after the change in ordering.

**Comment by dwaynebailey** [comment]
I think we also missed this as the exposure of Last Activity on the browse page seems to require really big screen real estate to be able to see it.  So test coverage in this area would be really important.
